backend/migrations/versions/2023_12_07_1451-e39a45a38d1f_add_test_execution_review_status_and_.py

```diff
@@ -0,0 +1,43 @@
+"""Add test execution review status and comment
+
+Revision ID: e39a45a38d1f
+Revises: 871eec26dc90
+Create Date: 2023-12-07 14:51:57.662163+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "e39a45a38d1f"
+down_revision = "871eec26dc90"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute(
+        "CREATE TYPE testexecutionreviewstatus AS "
+        "ENUM( "
+        "'UNDECIDED', "
+        "'MARKED_AS_FAILED', "
```

Comment:
```suggestion
        "'FAILED', "
```

Reply:
Or just `REJECTED`? Cause right now `TestExecution` already has a `status` field that can be `FAILED`. So having another `review_status` field that can also be `FAILED` would be confusing

Reply:
I also `REJECTED` would be better alternative to distinguish from other status types

Reply:
That makes sense.